### DISCUSSION TITLE
Using Menu inside a Modal

### DISCUSSION THREAD

**Discussion by farskid** [chakra-ui/chakra-ui#4459] [Q&A]
How would you use a `Menu` inside a `Modal` and keep the focus transferring and arrow key navigation working?
I have a `Menu` inside a `Modal` and I want to navigate between `MenuItem`s when Modal opens.
My current setup looks something like this:

```jsx
<Modal>
  <ModalContent>
    <Menu isOpen={true}>
      <MenuList>
        <MenuItem>Item 1</MenuItem>
        <MenuItem>Item 2</MenuItem>
      </MenuList>
     </Menu>
  </ModalContent>
</Modal>
```

The arrow key navigation is not working as it seems the focus is not transferred to the `MenuItem`s once `Modal` opens. anyone can set me in the right direction, please?

**Comment by 0xJosephX10** [comment]
Is your menu inside a portal. And also Modal is inside portal? If yes you prob need to remove the child one.
